pypesto/optimize/optimizer.py

```diff
@@ -558,3 +563,49 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
```

Comment:
Please add a link to the package home page here.

Reply:
I assume you mean PyPi?

Reply:
Any link works.

Reply:
Done